workers/ohsome_quality_analyst/indicators/mapping_saturation/metrics.py

```diff
@@ -1,3 +1,5 @@
+"""Collection of common error metrics as functions."""
+
```

Comment:
Just out of curiosity, why are we not relying on the functions provided by scikit-learn here? Because we want to avoid this addtional dependy?

Another question on the `mean error`: usually one would use the mean absolute error to evaluate the model performance.

Reply:
Yes, thats right. No need for an additional dependency since the calculation of those error metrics are pretty straightforward. Have a closer look at this file (`metrics.py`). The `mean error` is just one of the implemented errors. the `mean absolute error` is also implemented and currently used as criterion to choose the best model.

Reply:
ah perfect, yeah I missed the last lines of this file.